src/pageLayout/containers/TreeNav.tsx

```diff
@@ -1,8 +1,7 @@
 // Libraries
 import React, {FC, useContext, useEffect, MouseEvent} from 'react'
 import {Link, useLocation} from 'react-router-dom'
-import {useSelector} from 'react-redux'
-import {connect, ConnectedProps} from 'react-redux'
+import {useDispatch, useSelector} from 'react-redux'
 
```

Comment:
Right now `TreeNav` is using a mix of the `connect` and hooks apis from react-redux. 

Since we're already using `useSelector` here we might as well use `useDispatch` instead of the old map-dispatch-to-props and map-state-to-props paradigm.